koku/reporting/provider/ocp/models.py

```diff
@@ -767,3 +767,166 @@ class Meta:
         decimal_places=6,
         null=True
     )
+
+class OCPStorageLineItem(models.Model):
+    """Raw report storage data for OpenShift pods."""
+
+    class Meta:
+        """Meta for OCPStorageLineItem."""
+
+        unique_together = ('report', 'namespace', 'persistentvolumeclaim',)
+
+    id = models.BigAutoField(primary_key=True)
+
+    report_period = models.ForeignKey('OCPUsageReportPeriod',
+                                      on_delete=models.PROTECT)
+
+    report = models.ForeignKey('OCPUsageReport',
+                               on_delete=models.PROTECT)
+
+    # Kubernetes objects by convention have a max name length of 253 chars
+    namespace = models.CharField(max_length=253, null=False)
+
+    pod = models.CharField(max_length=253, null=True)
+
+    # Another node identifier used to tie the node to an EC2 instance
+    persistentvolumeclaim = models.CharField(max_length=253, null=True)
+
+    persistentvolume = models.CharField(max_length=253, null=True)
```

Comment:
```suggestion
    persistentvolume = models.CharField(max_length=253)
```

I think this shouldn't be nullable.

Reply:
good catch